src/selectors/prometheus/selectors.js

```diff
@@ -24,6 +24,11 @@ export const getUtilizationVectorStats = response => {
   return values && Array.isArray(values) ? values.map(timeValuePair => parseNumber(timeValuePair[1])) : null;
 };
 
+export const getUtilizationVectorTime = response => {
+  const values = get(response, 'data.result[0].values');
+  return values && Array.isArray(values) ? values.map(timeValuePair => parseNumber(timeValuePair[0] * 1000)) : null;
```

Comment:
Shouldn't it be `parseNumber(timeValuePair[0]) * 1000` ?

Reply:
`parseNumber` will just return the value given to it in the `Number` form after checking if its NaN, so `parseNumber(timeValuePair[0] * 1000))`  and  `parseNumber(timeValuePair[0]) * 1000` will probably do the same thing and hence I didn't notice this. If you feel the latter is better to understand, we can change it.